config/crd/kustomization.yaml

```diff
@@ -18,4 +18,5 @@ configurations:
   - kustomizeconfig.yaml
 
 patchesStrategicMerge:
-- crd-conversion-patch.yaml
+- patches/crd-conversion-patch-addresspools.yaml
+- patches/crd-conversion-patch-bgppeers.yaml
```

Comment:
why did you keep the StrategicMerge and not moved to `patches:` like the MetalLB commit?

Reply:
I kept it like that because I only wanted to fix the issue of splitting the yaml into two.
This is still going to work with `patchesStrategicMerge` and I prefer moving to `patches` will be done on a different PR, because we have that on 6 different kustomization files in the repo, and I want to keep this PR succinct.

Reply:
since this is already copying most of the other commit, I'd rather you just fix this line here as part of the alignment with metallb, regardless of the other places we need to fix the merge

Reply:
Done.